client/ayon_core/hosts/maya/api/pipeline.py

```diff
@@ -653,10 +653,6 @@ def on_task_changed():
             "Can't set project for new context because path does not exist: {}"
         ).format(workdir))
 
-    with lib.suspended_refresh():
```

Comment:
I think only @antirotor can tell if this is ok.

Reply:
The logic removed here was basically replaced by an artist facing prompt on file save to another context in the PR here https://github.com/ynput/ayon-core/pull/261